src/workspace/src/utils.ts

```diff
@@ -11,7 +12,7 @@ export const getSpotTileApps = async (): Promise<App[]> => {
     manifestType: "url",
     manifest: `${VITE_RT_URL}/fx-spot/${symbol}`,
     title: `${symbol} Spot Tile`,
-    icons: [],
+    icons: [{ src: ADAPTIVE_LOGO }],
```

Comment:
nice, I was hoping that it was as simple as this, once we found the icon in the types